.github/workflows/forward_model_load_check.yml

```diff
@@ -74,9 +74,10 @@ jobs:
       - uses: actions/download-artifact@v1
         with:
           name: vw_generated_models
-          path: vw_generated_models
+          path: .vw_runtests_model_gen_working_dir/test_models
```

Comment:
default path here is the working directory and tilde doesn't work in placing it in path, hence the `mv` below